app/views_redesigned/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render "devise/layout" do %>
+  <%= simple_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { class: "auth-form js-validate mx-auto", method: :post }) do |f| %>
```

Comment:
not sure we need this `js-validate` class. May want to check on that but I think that's from the old template.

Reply:
:heavy_check_mark: